Library/Homebrew/dev-cmd/test-bot.rb

```diff
@@ -961,7 +961,7 @@ def test_bot
     # Tap repository if required, this is done before everything else
     # because Formula parsing and/or git commit hash lookup depends on it.
     # At the same time, make sure Tap is not a shallow clone.
-    # bottle revision and bottle upload rely on full clone.
+    # rebuild and bottle upload rely on full clone.
```

Comment:
I think this could be a tad clearer if it was “Bottle rebuild” instead of just “rebuild”.